pages/glossary/owned-by.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Owned by
+key: owned-by
+---
+
+An element A is owned by element B if element A is a child of element B in the [accessibility tree](https://www.w3.org/TR/act-rules-aspects/#input-aspects-accessibility).
```

Comment:
I dig this 👍

Reply:
Is it relevant to use the definition of descendant? This is used in #979 

https://dom.spec.whatwg.org/#concept-tree-descendant

Reply:
I don't think so. The accessibility tree has no spec, so I'm not entirely sure a DOM definition would be appropriate here.

Reply:
I would say that "descendant" is a common an well defined term when it's about trees. In the same way that "child" is common and well defined for trees.

I think we can write "A is a descendant of B in the accessibility tree" without further definition or link. (either that or I'm going to challenge the use of "child" without definition 😈).
OTOH, it seems that "child" instead of "descendant" is intentional in this definition (Failed Examples 2 and 3 are definitely using "child" in their use of "owned by").